graveyard.json

```diff
@@ -16,8 +16,8 @@
     "type": "service"
   },
   {
-    "dateClose": "1998-10-02",
-    "dateOpen": "2010-07-07",
+    "dateClose": "2010-07-07",
+    "dateOpen": "1998-10-02",
```

Comment:
The YouTube Leanback dates were changed. YouTube did not exist in 1998. 😄

See the current `graveyard.json` file for the correct dates.

https://github.com/codyogden/killedbygoogle/blob/4892edb21509d4660f0ee03f79c780062da17cd9/graveyard.json#L10-L17